unpackaged/post/first/translations/de.translation

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
@codexmarcel Because the new round of translations include translations in the `unpackaged/config/post` folder, this may create problems if this is deployed into an org without translation workbench enabled. For example, the "beta", "trial", "release" and many other scratch org definitions do not have translation workbench enabled. As a result, running `deploy_post` task fails against these orgs.

I'm a bit uncertain on where else this task may be used. If it's pretty limited to just our own configuration we may be able to modify the scratch org definitions. However, if this is used in many other areas, it may be necessary to remove these translations from the `post/first` folder. I suppose they could be included in the main trial org configuration since the trial config is always deployed after `post/first`.
cc @davidmreed

Reply:
Since the pre and post folders both had translations in master, I'm re-running the install_beta task one more time from this branch to make sure that it still fails.

Reply:
The biggest issue I see here is that `unpackaged/post` is also deployed in _both_ NPSP installers - the primary customer-facing one and the one for Trailhead Playgrounds. I don't think it is safe to make the assumption that those orgs have Translation Workbench turned on.

We don't currently have a preflight check for Translation Workbench. If we did (I don't know at present how to detect it!) we could break out these translations into a separate task and conditionally deploy them if and only if Translation Workbench were turned on.

What's the objective for these translations look like? Are they primarily supporting the TSO, or were we aiming to deliver them via MetaDeploy as well?

Reply:
Good questions. Pre and Post were included in the translations because they both have unpackaged metadata. In particular, the `pre/account_record_types/` folder has the Household Account and Organization record types. It is odd that the pre folders already had the translation files. The only change in that folder structure is that the customObjectTranslation block was added to the package.xml.

I just confirmed that `install_beta` fails on the  `unpackaged/post/first` deployment step due to Translation Workbench not being installed. I suspect that the metadata api allows the deployment of the object translations in the `unpackaged/pre` folder, but is failing on the global (quickAction) translations in the post folder. That seems to be the only difference. I think the best solution - at least for now - is to just remove these two translations from the post config folder. They're very minor, and we've never had them in orgs.